tests/integration/tables/helper.cpp

```diff
@@ -203,6 +203,20 @@ bool is_valid_hex(const std::string& value) {
 }
 
 bool validate_value_using_flags(const std::string& value, int flags) {
+  // Early return on EmptyOk
```

Comment:
Should we assert that `EmptyOk` and `NonEmpty` are not both set?

Reply:
Not a bad optimization. I didn't think of it